include/seeds.accounts.hpp

```diff
@@ -26,6 +26,7 @@ CONTRACT accounts : public contract {
           vouches(receiver, receiver.value),
           vouchtotals(receiver, receiver.value),
           reqvouch(receiver, receiver.value),
+          flags(receiver, receiver.value),
```

Comment:
These changes are unrelated and should not be here

Ways to fix
A - merge master into this branch, should fix it , but if not
B - create new branch off master, only add readme updates, nothing else, create new PR

Reply:
updated merge